### PR TITLE
Reduce logging to sentry

### PR DESCRIPTION
**Pull request by dojutsu-user**
Related Issues - #4978, #3856, #4590,

### CODE REVIEW

**Comment by dojutsu-user** [comment]
Are the exceptions defined in `readthedocs.doc_builder.exceptions` and `readthedocs.projects.exceptions` are to be exempted from the logging in the same way?

File: readthedocs/settings/base.py [hunk 1]
```diff
@@ -266,6 +266,9 @@ def USE_PROMOS(self):  # noqa
         },
     }
 
+    # Sentry
+    SENTRY_CELERY_IGNORE_EXPECTED = True
```
Comment (stsewd) [inline readthedocs/settings/base.py]: This setting comes from https://docs.sentry.io/clients/python/integrations/django/#additional-settings (just linking for reference)

Comment (dojutsu-user) [reply readthedocs/settings/base.py]: Thank you.
I should have added this in the description.

**Comment by ericholscher** [comment]
Looks good. Thanks!

**Comment by humitos** [comment]
I suppose that we should remove the logic that I wrote some time ago if we are going in this direction:

https://github.com/rtfd/readthedocs.org/blob/77f9d8720a82a6d5ef2295795297dd7df6cbc37b/readthedocs/doc_builder/environments.py#L477-L486

**Comment by humitos** [comment]
Or, at least, we should think about what to do with them. Also, does Celery provides a way to "log them as WARNING" or it does log them but do not send them to Sentry only?

**Comment by dojutsu-user** [comment]
With ref to [comment](https://github.com/rtfd/readthedocs.org/pull/5054#issuecomment-451084858):
Celery does log them but as `INFO`.


I tried making a simple task with celery and using `task.throws`.
I used this example

```python3
# tasks.py
from celery import Celery

app = Celery('tasks', broker='pyamqp://guest@localhost//')

@app.task(throws=(ZeroDivisionError,))
def divide(x, y):
    if y == 0:
        raise ZeroDivisionError("Divide by zero")
    return x/y
```

And when i tried `add.delay(4,0)`, i get this:
```
[2019-01-03 15:06:08,893: INFO/MainProcess] Received task: tasks.divide[eadc9b77-208b-460b-8cce-a891812a5895]  
[2019-01-03 15:06:08,897: INFO/ForkPoolWorker-2] Task tasks.divide[eadc9b77-208b-460b-8cce-a891812a5895] raised expected: ZeroDivisionError('Divide by zero',)
```

**Pull request merged**